SystemServices/CHANGELOG.md

```diff
@@ -5,10 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2022-08-24
```

Comment:
Pls retain the same version as main branch [1.0.2] so that we can tie the version to this change. I understand this will not do version updates in sequence but it will eventually catch up (once OTA_6 gets branched from stable2)

Reply:
Done